drivers/SmartThings/matter-switch/src/aqara-cube/init.lua

```diff
@@ -0,0 +1,217 @@
+local capabilities = require "st.capabilities"
+local clusters = require "st.matter.generated.zap_clusters"
+local device_lib = require "st.device"
+
+local cubeAction = capabilities["stse.cubeAction"]
+local cubeFace = capabilities["stse.cubeFace"]
+
+local COMPONENT_TO_ENDPOINT_MAP = "__component_to_endpoint_map"
```

Comment:
When the button driver was brought into the switch driver, a new field was created for the component to endpoint map for button devices. `__component_to_endpoint_map` still exists for legacy switch devices that still use MCD, but will not be used for new switch devices. Could you please update this to use the new field introduced for button MCD devices, `__component_to_endpoint_map_button`?

Reply:
Thanks, I updated it.